src/thematic/0_generic/sfp-001_vervanging-terrest_nl/03_beperkingen.Rmd

```diff
@@ -0,0 +1,10 @@
+## Beperkingen tot het protocol (indien van toepassing)
+
+<!--
+Bespreek hieronder onder welke condities het veldprotocol niet gebruikt kan worden of wanneer afwijkingen nodig zijn ten opzichte van het standaard protocol. Vermeld hier ook mogelijke interferenties, knelpunten, …
+-->
+
+*Minder geschikt voor meetnetten voor habitattypen die vooral uit grote polygonen bestaan, zoals heide?*
+*Niet geschikt voor meetnetten voor habitattypen die niet goed toegankelijk zijn of weinig overzichtelijk zijn, zoals bossen?*
```

Comment:
Ook de grote of onoverzichtelijke polygonen zullen voor centroïdgekoppelde types (= de meeste terrestrische) moeten leiden tot een steekproefeenheid. De essentie is dat het steekproefkader bijgewerkt raakt zodat alsnog een ruimtelijk gebalanceerde trekking uit de polygoon gebeurt. Het kan gericht gebeuren (punten bezoeken) volgens de ruimtelijk gebalanceerde puntenvolgorde.

Een alternatief is nadere kartering om de polygoon op te splitsen tot deelpolygonen die de verspreiding van de types beter in beeld brengen, zeker als het doeltype vooral aan één zijde voorkomt bv. Mogelijks is het niet de bedoeling om dit soort lokale karteerinformatie in de _habitatkaart_ te integreren, maar het is wel informatie die we willen om het steekproefkader (lokaal) te verbeteren.

Reply:
OK. Verwijderd.